package.json

```diff
@@ -3,34 +3,8 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "start": "cross-env MODE=development CONFIG=dev webpack serve --open",
-    "build": "webpack",
-    "build:as-development": "cross-env MODE=development CONFIG=as-development npm run build",
-    "build:test-staging": "cross-env MODE=production CONFIG=test-staging npm run build",
-    "build:as-staging": "cross-env MODE=production CONFIG=beta npm run build",
-    "build:zm-staging": "cross-env MODE=production CONFIG=zm-staging npm run build",
-    "build:zm-production": "cross-env MODE=production CONFIG=zm-production npm run build",
-    "build:za-staging": "cross-env MODE=production CONFIG=za-staging npm run build",
-    "build:za-production": "cross-env MODE=production CONFIG=za-production npm run build",
-    "build:et-staging": "cross-env MODE=production CONFIG=et-staging npm run build",
-    "build:et-production": "cross-env MODE=production CONFIG=et-production npm run build",
-    "build:mw-staging": "cross-env MODE=production CONFIG=mw-staging npm run build",
-    "build:mw-production": "cross-env MODE=production CONFIG=mw-production npm run build",
-    "build:jm-staging": "cross-env MODE=production CONFIG=jm-staging npm run build",
-    "build:jm-production": "cross-env MODE=production CONFIG=jm-production npm run build",
-    "build:ca-staging": "cross-env MODE=production CONFIG=ca-staging npm run build",
-    "build:uk-staging": "cross-env MODE=production CONFIG=uk-staging npm run build",
-    "build:ro-staging": "cross-env MODE=production CONFIG=ro-staging npm run build",
-    "build:hu-staging": "cross-env MODE=production CONFIG=hu-staging npm run build",
-    "build:hu-production": "cross-env MODE=production CONFIG=hu-production npm run build",
-    "build:e2e-development": "cross-env MODE=development CONFIG=e2e-development npm run build",
-    "build:co-staging": "cross-env MODE=production CONFIG=co-staging npm run build",
-    "build:co-production": "cross-env MODE=production CONFIG=co-production npm run build",
-    "build:cl-staging": "cross-env MODE=production CONFIG=cl-staging npm run build",
-    "build:zw-staging": "cross-env MODE=production CONFIG=zw-staging npm run build",
-    "build:pl-staging": "cross-env MODE=production CONFIG=pl-staging npm run build",
-    "build:mt-staging": "cross-env MODE=production CONFIG=mt-staging npm run build",
-    "build:mt-production": "cross-env MODE=production CONFIG=mt-production npm run build",
+    "start": "cross-env MODE=development CONFIG=as-development webpack serve --open",
+    "build": "cross-env webpack",
```

Comment:
Unneeded `cross-env`?

Reply:
I think this one is needed for @stephenhand running local commands?

Reply:
The build script isn't setting any env vars, so I don't *think* it needs `cross-env` but I could be wrong.